README.md

```diff
@@ -0,0 +1,15 @@
+# Instruction togit nub
+
+##  Create local repo
+**git init**  создание локального репозитория в проектной папке.  
```

Comment:
you also can use \`code\` to add one line `code` block.  \` - is tilde (~ button).
or multi line code with triple \`\`\`
```
example
example
```

It is more appropriate for commands that are shown here rather than **bald** text.